docs/source/quantization/auto_awq.rst

```diff
@@ -5,21 +5,19 @@ AutoAWQ
 
 .. warning::
 
-   Please note that AWQ support in vLLM is under-optimized at the moment. We would recommend using the unquantized version of the model for better
-   accuracy and higher throughput. Currently, you can use AWQ as a way to reduce memory footprint. As of now, it is more suitable for low latency
-   inference with small number of concurrent requests. vLLM's AWQ implementation have lower throughput than unquantized version.
+   Please note that Marlin kernels are up to 2.59x faster than the AWQ GEMM kernel which is under-optimized in vLLM. You can use the faster kernels by passing setting the `quantization` parameter in vLLM to ``awq_marlin`` instead of ``awq``.
```

Comment:
Given the above, I would just remove mention of the `quantization` arg entirely to let the automatic conversion happen.
```suggestion
```